src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2227,26 +2227,33 @@ namespace cryptonote
       uint64_t from_height;
       uint64_t to_height;
       bool cumulative;
+      bool binary;
```

Comment:
Maybe technically these changes require bumping the RPC version?

Reply:
Sigh. And I'm the one who added the version number... It's so easy to forget :/